batch/src/test/java/org/frankframework/batch/BatchFileTransformerPipeTest.java

```diff
@@ -0,0 +1,54 @@
+package org.frankframework.batch;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import org.frankframework.core.PipeRunResult;
+import org.frankframework.pipes.PipeTestBase;
+import org.frankframework.stream.Message;
+import org.frankframework.testutil.TestFileUtils;
+
+class BatchFileTransformerPipeTest extends PipeTestBase<BatchFileTransformerPipe> {
+
+	@Override
+	public BatchFileTransformerPipe createPipe() {
+		return new BatchFileTransformerPipe();
+	}
+
+	@TempDir
+	private Path tempDir;
+
+	@Test // Not really using the full features of the pipe, just testing some flows
+	void doPipeMovesFileAfterProcessing() throws Exception {
+		URL inputUrl = TestFileUtils.getTestFileURL("/input.xml");
+		File file = new File(inputUrl.toURI());
+		Message input = new Message(file.getAbsolutePath());
+
+		pipe.setDelete(false);
+		pipe.configure();
+		pipe.start();
+
+		PipeRunResult pipeRunResult = pipe.doPipe(input, session);
+		assertTrue(pipeRunResult.getResult().isEmpty());
+		assertTrue(file.exists());
```

Comment:
Ik heb geprobeerd hier iets meer uit te halen, qua testen of testvalidatie, maar dat bleek lastig. In ieder geval dekken we nu een basic flow. Voor nieuwe code is dit natuurlijk te weinig, maar voor oude en niet echt meer onderhouden code lijkt me dit een goede start.

Reply:
> Ik heb geprobeerd hier iets meer uit te halen, qua testen of testvalidatie, maar dat bleek lastig.

Kan je in de code ook een kort comment zetten van die strekking? Dan weten we voor later tenminste waarom de tests niet veel testen 😬

Reply:
Ik had op regel 30 al zoiets staan. Ik zet het wel in Javadoc boven de test class. Dan is het nog duidelijker.